data/web/inc/clientconfig.inc.php

```diff
@@ -0,0 +1,143 @@
+<?php
+require_once 'prerequisites.inc.php';
+require_once 'vars.inc.php';
+
+$client_config = array(
+     'useEASforOutlook' => 'yes',
+     'imap' => array(
+       'server' => $mailcow_hostname,
+       'port' => '993',
+       'ssl' => 'on',
+       'tlsport' => '143',
+     ),
+     'pop3' => array(
+       'server' => $mailcow_hostname,
+       'port' => '995',
+       'ssl' => 'on',
+       'tlsport' => '110',
+     ),
+     'smtp' => array(
+       'server' => $mailcow_hostname,
+       'port' => '465',
+       'ssl' => 'on',
+       'tlsport' => '587',
+     ),
+     'sogo' => array(
+       'server' => $mailcow_hostname,
+       'port' => '443',
+       'ssl' => 'on',
+     ),
+     'http' => array(
+       'server' => $mailcow_hostname,
+       'port' => '443',
+       'ssl' => 'on',
+     ),
+     'activesync' => array(
+       'url' => 'https://'.$mailcow_hostname.'/Microsoft-Server-ActiveSync',
+     ),
+);
+
+function get_client_config() {
+  global $pdo, $client_config;
+  $config = $client_config;
+  
+  // use the SRV records of the first domain to obtain the correct port numbers
```

Comment:
The default values are not overwritten if the SRV records don't exist. They are already in the `$config` array.